e2e-pw/src/oss/poms/action-row/display-options.ts

```diff
@@ -0,0 +1,32 @@
+import { Locator, Page } from "src/oss/fixtures";
+
+type SidebarStatisticsMode = "slice" | "group";
+type SidebarMode = "fast" | "best" | "all";
+type SidebarSortMode = "count" | "value";
+
+export class DisplayOptionsPom {
+  readonly page: Page;
+
+  readonly grid: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async setSidebarStatisticsMode(mode: SidebarStatisticsMode) {
+    const selector = this.page.getByTestId(
+      `tab-option-View ${mode} sidebar statistics`
+    );
+    return selector.click();
```

Comment:
do we need `await` here? 🤔

Reply:
We could `await` or just return the promise and expect the caller to await.